src/index.tsx

```diff
@@ -75,13 +76,15 @@ createRoot(container).render(
           <Web3Provider>
             <VioletProvider client={client}>
               <ApolloProvider client={apolloClient}>
-                <BlockNumberProvider>
-                  <Updaters />
-                  <ThemeProvider>
-                    <ThemedGlobalStyle />
-                    <App />
-                  </ThemeProvider>
-                </BlockNumberProvider>
+                <AnalyticsProvider>
+                  <BlockNumberProvider>
+                    <Updaters />
+                    <ThemeProvider>
+                      <ThemedGlobalStyle />
```

Comment:
At this point, I'd consider a making a component to wrap all the provider all together.

Reply:
That's certainly something we should do, but with better care than bundling it on this PR bc of the props and which ones we want here vs defined on that component, I believe that deserves a ticket to itself

Reply:
might be worth opening it now just to keep track of the idea

Reply:
https://violetprotocol.atlassian.net/browse/VIOL-1020